src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -3149,7 +3185,7 @@ void ApplicationManagerImpl::AllowStreaming(uint32_t app_id) {
   LOG4CXX_AUTO_TRACE(logger_);
 
   ApplicationSharedPtr app = application(app_id);
-  if (!app || !app->is_navi()) {
+  if (!app || (!app->is_navi() && !app->mobile_projection_enabled())) {
     LOG4CXX_ERROR(logger_, "Navi application not found");
```

Comment:
As a side note, these log statements should probably refer to both Navi and Projection apps